app/harvesters/uva_harvester.rb

```diff
@@ -0,0 +1,153 @@
+##
+# A harvester implementation for UVA
+#
+class UVAHarvester
+  include Krikri::Harvester
+
+  DEFAULT_THREAD_COUNT = 10
+  DEFAULT_MAX_RECORDS = 0
+
+  ##
+  # Initialize, and set default options as appropriate.
+  #
+  # @param opts [Hash] a hash of options as defined by {.expected_opts}
+  #
+  # @example
+  #    Typical instantiation, good for most cases:
+  #
+  #      UVAHarvester.new(uri: 'http://example.edu/fedora/...')
+  #
+  # Accepts options passed as `:uva => opts`
+  #
+  # Options allowed are:
+  #
+  #   - threads:     The number of records to fetch asynchronously
+  #                  in a batch (default: 10)
+  #   - name:        See Krikri::Harvester#initialize.  Defaults to "uva"
```

Comment:
Looks like this default is not actually getting set, unless I'm missing something.